RELEASENOTES.md

```diff
@@ -11,3 +11,4 @@
 ## Internals
 - Fixed the Dockerfile used when testing PRs. ([#1057](https://github.com/realm/realm-studio/pull/1057))
 - Removing all existing and future unused locals. ([#1058](https://github.com/realm/realm-studio/pull/1058))
+- Ensuring users open partial Realms in a read-only mode. ([#1062](https://github.com/realm/realm-studio/pull/1062))
```

Comment:
This has a public effect so shouldn't just be under "internals"?

Reply:
You're right - moving this to the bugfix section.